AnnotationModule.py

```diff
@@ -0,0 +1,63 @@
+import csv
+import xml.etree.ElementTree as ET
+import cv2
+import numpy as np
+import openslide
+import os
+from skimage import io
+from BaseImage import printMaskHelper
+from PIL import Image, ImageDraw
+

```

Comment:
your module has essentially at least doubled the # of dependencies that HistoQC requires. this is non-obvious, and many of these are not installed by default (e.g., xml). while i would prefer a leaner version, if this is what has to happen, then so be it. since the modules themselves are loaded dynamically when included in the pipeline, users which don't use your module won't have these requirements. as a result, when adding dependencies you need to modify the documentation (e.g., Readme.md in this case) to state what they are. In this case, i would add a sub section below the current one with the name of your module and state its unique additional dependenices, so users who don't need it won't need to meet the requirements